odoo/addons/base/models/res_partner.py

```diff
@@ -33,9 +33,32 @@
 def _lang_get(self):
     return self.env['res.lang'].get_installed()
 
+def timezone_offset(tz):
+    """Take the timezone in a string format and return the hour offset.
+
+    e.g.
+        'Africa/Cairo' -> '+0200'
+    """
+    utcnow = datetime.datetime.utcnow()
+    localized_datetime = pytz.timezone(tz).localize(utcnow)
+    return localized_datetime.strftime('%z')
+
+def format_timezone(tz):
```

Comment:
I would move those two small tools directly in tools/misc.py along with other format methods (format_date, ...) . That way we keep formatting methods grouped.